awx/main/tasks.py

```diff
@@ -679,6 +685,48 @@ def update_host_smart_inventory_memberships():
         smart_inventory.update_computed_fields()
 
 
+@task(queue=get_local_queuename)
+def migrate_legacy_event_data(tblname):
+    if 'event' not in tblname:
+        return
+    with advisory_lock(f'bigint_migration_{tblname}', wait=False) as acquired:
+        if acquired is False:
+            return
+        chunk = 1000000
```

Comment:
Make this so support can edit it if a customer migration fails. I'm thinking of the case where Postgres gets overloaded in some way by the massive data-move size because the users job events are large.